### PR TITLE
Added callout on personal access token

### PR DESCRIPTION
**Pull request by dancer-yuma**
## What I changed

- added a callout on personal access token to be used for password authentication in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,6 +167,14 @@ Run the following command:
 
 Now, navigate to your GitHub repository. The changes should be successfully pushed!
 
+> You may be asked to enter password and encounter the following error. To fix this, you have to use personal access token for password authentication. More information is available on [the official document](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic).
```
Comment (abolinsky) [inline README.md:170]: You may be asked to enter your GitHub account username and password and encounter the following error.

```sh
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/ms1-learner/cpp-01-en.git/'
```

To fix this, you must use a personal access token for authentication. Information detailing this process is available in [the official GitHub documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic).

**Review by abolinsky — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lordmadi) [reply README.md:170]: I guess the flow Alex suggested is more readable. Can you change it @dancer-yuma?

Comment (dancer-yuma) [reply README.md:170]: Yes! Thank you @abolinsky and @lordmadi! Working on it.

Comment (dancer-yuma) [reply README.md]: Resolved in [1288b3b](https://github.com/ms1-learner/cpp-01-en/pull/9/commits/1288b3b5fb2f1da809fe0f0ba159110155813c56).

**Review by lordmadi — APPROVED** [review]
lgtm

**Review by lordmadi — APPROVED** [review]

**Review by abolinsky — APPROVED** [review]

**Pull request merged**